src/appshell/view/dockwindow/internal/dockbase.h

```diff
@@ -102,8 +108,13 @@ class DockBase : public QQuickItem
 
     QRect frameGeometry() const;
 
+    bool isInOneFrame(const DockBase* other) const;
+    void setFramePanelOrder(int order);
+
     Q_INVOKABLE void resize(int width, int height);
 
+    ui::NavigationPanel* contentNavigationPanel();
```

Comment:
ui::NavigationPanel* contentNavigationPanel() **const**;

Reply:
This shouldn't be constant because we are changing the order of the panels in DockPageView